nixos/modules/hardware/bitcraze.nix

```
@@ -0,0 +1,23 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.bitcraze;
+
+in
+{
```

nit: what's up with the formatting here?

```suggestion
{ config, lib, pkgs, ... }:
let
  cfg = config.hardware.bitcraze;
in
{
```